src/Util/TemplateLinter.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Util;
+
+use Symfony\Bundle\MakerBundle\Exception\RuntimeCommandException;
+use Symfony\Component\Process\Process;
+
+/**
+ * Linters used by make:* commands to cleanup the generated files.
+ *
+ * @author Jesse Rushlow <jr@rushlow.dev>
+ *
+ * @internal
+ */
+final class TemplateLinter
+{
+    public function __construct(
+        private ?string $phpCsFixerBinaryPath = null,
+        private ?string $phpCsFixerConfigPath = null,
+    ) {
+        $this->setConfig();
+        $this->setBinary();
+    }
+
+    public function lintPhpTemplate(string|array $templateFilePath): void
+    {
+        if (\is_string($templateFilePath)) {
+            $templateFilePath = [$templateFilePath];
+        }
+
+        foreach ($templateFilePath as $filePath) {
+            $process = Process::fromShellCommandline(sprintf('php %s --config=%s --using-cache=no fix %s', $this->phpCsFixerBinaryPath, $this->phpCsFixerConfigPath, $filePath));
+            $process->run();
+        }
+    }
+
+    private function setBinary(): void
+    {
+        if (null !== $this->phpCsFixerBinaryPath) {
+            $this->checkPathExists($this->phpCsFixerBinaryPath, false);
+
+            return;
+        }
+
+        $defaultBinaryPath = 'bin/php-cs-fixer';
+
+        if (file_exists($defaultBinaryPath)) {
+            $this->phpCsFixerBinaryPath = $defaultBinaryPath;
+
+            return;
+        }
+
+        $this->phpCsFixerBinaryPath = \dirname(__DIR__).'/Bin/php-cs-fixer-v3.13.0.phar';
```

Comment:
Any thoughts on how we could not forget to routinely update this?

Reply:
![post-it](https://user-images.githubusercontent.com/40327885/206539637-995baae2-5384-46a3-a069-4fae82e01374.jpg)

Reply:
Aside from post it notes, thinking we have a new ci job the runs daily to check for the latest version and compares it to our `php-cs-fixer.VERSION.php` bin file name... I don't think we want this running on every PR/push as that we result in a bunch of rebases over time?

Reply:
LOL

Maybe a monthly CI job would be awesome if it's not too much work - one that could create a PR with the change would be ideal.